packages/reactivity/src/reactive.ts

```diff
@@ -217,13 +278,26 @@ function createReactiveObject(
   return proxy
 }
 
+/**
+ * Checks if an object is a reactive proxy created by `reactive`.
+ *
+ * It also returns `true` if the given object was created by `readonly`, but is
+ * wrapping another proxy created by `reactive`.
```

Comment:
I think we could just remove this and add more details on the previous sentence.